src/components/QueryBuilder.vue

```diff
@@ -23,6 +23,15 @@
 		<div class="querybuilder__run">
 			<Button @click.native="runQuery" type="primaryProgressive">Run query</Button>
 		</div>
+		<div class="querybuilder__result">
+			<div v-if="!showQueryResults" class="querybuilder__result__description">Results will be displayed here</div>
+			<iframe v-bind:src="'https://query.wikidata.org/embed.html#' + encodedQuery"
+			class="querybuilder__result__iframe"
+			referrerpolicy="origin"
+			v-if="showQueryResults"
+			sandbox="allow-scripts allow-same-origin allow-popups">
```

Comment:
Do we need "allow-popups"?

Reply:
we don't. 
tested it and removed it. thanks for the tip